### PR TITLE
Mobile first

### PR DESCRIPTION
**Pull request by joyapisi**
# What I Have Done:
`Create First Section of the Mobile Portfolio Website `
-Use Figma template to design the first page of the Portfolio website for the mobile version
-Create the toolbar (header) of the Portfolio website for the mobile version
-Create the headline section of the Portfolio website for the mobile version
-Use flexbox to place elements on the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap">
+        <title>Portfolio</title>
+</head>
+
+<body>
+  
+  <header>
+    <nav title="navigation-bar">
+      <button class="nav-button1">
+        <span class="logo">Joy</span>
```
Comment (wayungi) [inline index.html:17]: - [ ] Please make this logo a link tag that navigates back to the home page. This is for three reasons: convention, semantics, and user experience(UX). 
[Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.😃

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <li><a class="navigationList" href="#about">About</a></li>
+        <li><a class="navigationList" href="#contactme">Contact</a></li>
+      </ul>
+    
+      <div class="nav-button2">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
```
Comment (wayungi) [inline index.html:31]: - [OPTIONAL] Please only include navigation elements in the nav tag. You  nav tag also has the logo in it. What do you think about takin it out? 🤝

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<section class="mobilemain" title="Joy's-Portfolio-Home-page">
+ <article class="headline">
+   <h1 id="heading">I’m Joy<br>
+     Glad to see you!</h1>
+   
+   <p id="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+   <div class="smedia" title="social-media-accounts">
+   
+   <h5 id="heading5">LET’S CONNECT</h5>
```
Comment (wayungi) [inline index.html:44]: - [ ] According to the design, this should be a button or a link. Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href. Please change it to a button/link. If you decide to make it a button, remember to add the button styling: Active, Hover and Pressed (see design)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+.mobilemain {
+  position: relative;
+  width: 375px;
+  height: 810px;
+  background: #fff;
+  right: 0;
+  overflow-x: hidden;
+}
+
```
Comment (wayungi) [inline style.css:125]: - [ ] Please align the content inside this section with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. 👍🏾

File: index.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+
+<!--Page 1-->
+<section class="mobilemain" title="Joy's-Portfolio-Home-page">
+ <article class="headline">
```
Comment (wayungi) [inline index.html:36]: - [OPTIONAL ] It is not necessary to have only one article inside a section tag as in this case. What do you think about removing the article and leaving all the content inside the `section` tag? 🤔

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @joyapisi , 

You have done a very great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
- [ ] Good README
- [ ] Good commit messages
- [ ] Linter checks passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joyapisi** [comment]
What I Have Done:
`corrections from code reviewer`
-Update README.md file
-Update HTML semantics
-Update CSS styling

File: index.html [hunk 6]
```diff
@@ -12,11 +12,12 @@
 <body>
   
   <header>
+    <button class="nav-button1"> 
+      <a href="#about"></a>
+      <span class="logo">Joy</span>
```
Comment (zieeco) [inline index.html:17]: - Hey, great job, however, you need to wrap the logo inside the anchor tag to make it a link. please, fix this issue as suggested by the previous reviewer.

File: index.html [hunk 7]
```diff
@@ -41,7 +42,7 @@ <h1 id="heading">I’m Joy<br>
 
    <div class="smedia" title="social-media-accounts">
    
-   <h5 id="heading5">LET’S CONNECT</h5>
+   <h5> <a id="heading5" href="#">LET’S CONNECT </a></h5>
```
Comment (zieeco) [inline index.html:45]: - Please, kindly remove the h5 and leave it as a link as pointed out by the previous reviewer.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @joyapisi ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (joyapisi) [reply index.html:17]: Thank you. Kindly help me know: Does this mean I will also remove the <button> tag to be left with only the <a> tag?

**Comment by joyapisi** [comment]
# What I Have Done✍🏽:
`A correction from code reviewer`
-Update the logo in the HTML file to be inside an 'a' tag instead of a button.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**